test/e2e/specs/specs-i18n/wp-i18n__logged-out-redirect.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @group i18n
+ */
+
+import { setupHooks, BrowserHelper } from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+const WPCOM_HOMEPAGE = 'https://wordpress.com';
```

Comment:
For more flexibility, I would use `DataHelper.getCalypsoURL()` here!

Reply:
I avoided using it, because the method name made it seem a bit confusing to use it for a non-Calypso test. However, I agree that it could be better than having the base URL hardcoded.

Updated in https://github.com/Automattic/wp-calypso/pull/57707/commits/4672e76befb6a1f561b7b8ca8ba7ff8849101301#diff-28f0ec8f4f0ff478f3f0b7652a3e70e9dfb19f2fd412f6d855d83da289b02785R18-R22.